lib/vericite.rb

```diff
@@ -0,0 +1,460 @@
+#
```

Comment:
This file looks like a copy of the turnitin client with modifications to call the vericite gem code.  This probably should be redone to match the vericite way of operating.  The turnitin code is generally frowned upon, but was left as is under 1) it ain't broke and 2) it's going away.  Since #2 isn't true in this case, we'd like to see a better attempt at this file.